RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -87,7 +87,7 @@
     minPt = 0.2,
 )
 highPtTripletStepTrajectoryFilterBase = _highPtTripletStepTrajectoryFilterBase.clone(
-    maxCCCLostHits = 2,
+    maxCCCLostHits = 0,
     minGoodStripCharge = dict(refToPSet_ = 'SiStripClusterChargeCutLoose')
```

Comment:
Yes, intended. `HighPtTripletStep` (and `LowPtQuadStep` and `DetachedQuadStep`) are used only in phase1 and 2, and hopefully we don't need (this) mitigation for them.